ykcapi/src/lib.rs

```diff
@@ -95,29 +100,66 @@ impl Registers {
     }
 }
 
+#[derive(Debug)]
+#[repr(C)]
+struct AOTVar {
+    bbidx: u32,
```

Comment:
Are these types always `u32` no matter the platform?

Reply:
Good point. We are actually using `size_t` in `jitmodbuilder.cc` but since LLVM doesn't have a `size_t` type, we probably need to add platform specific code when passing these values into the deoptimisation call.